fission/main.go

```diff
@@ -74,13 +78,13 @@ func main() {
 	htUrlFlag := cli.StringFlag{Name: "url", Usage: "URL pattern (See gorilla/mux supported patterns)"}
 
 	// Resource & scale related flags (Used in env and function)
-	minCpu := cli.StringFlag{Name: "mincpu", Usage: "Minimum CPU to be assigned to pod (In millicore, minimum 1)"}
-	maxCpu := cli.StringFlag{Name: "maxcpu", Usage: "Maximum CPU to be assigned to pod (In millicore, minimum 1)"}
-	minMem := cli.StringFlag{Name: "minmemory", Usage: "Minimum memory to be assigned to pod (In megabyte)"}
-	maxMem := cli.StringFlag{Name: "maxmemory", Usage: "Maximum memory to be assigned to pod (In megabyte)"}
-	minScale := cli.StringFlag{Name: "minscale", Usage: "Minimum number of pods (Uses resource inputs to configure HPA)"}
-	maxScale := cli.StringFlag{Name: "maxscale", Usage: "Maximum number of pods (Uses resource inputs to configure HPA)"}
-	targetcpu := cli.IntFlag{Name: "targetcpu", Value: 80, Usage: "Target average CPU usage percentage across pods for scaling"}
+	minCpu := cli.IntFlag{Name: "mincpu", Usage: "Minimum CPU to be assigned to pod (In millicore, minimum 1)"}
+	maxCpu := cli.IntFlag{Name: "maxcpu", Usage: "Maximum CPU to be assigned to pod (In millicore, minimum 1)"}
+	minMem := cli.IntFlag{Name: "minmemory", Usage: "Minimum memory to be assigned to pod (In megabyte)"}
```

Comment:
Hmm, I have a slight preference for the strings because the user can specify the units instead of checking the help to see which units are needed.  Also if no units are specified you could always default to megabytes.

Reply:
I think we should use IntFalg instead of StringFlag for couple reasons
1. We use `Int` to get value internally. And since the flags are public for a long time, if we change the input format from number to string it may break users code.
https://github.com/fission/fission/blob/b456dec1385838279a0917b6542b9e0416a6d796/fission/environment.go#L280-L285
2. For most of users, they don't actually know what is the correct unit to specify with, also kubernetes provides to many way to specify the resource unit and may make more confusion to users. For example, cpu can be represent as`100m` or `0.1` while memory is `Ei, Pi, Ti, Gi, Mi, Ki`. It also brings difficulity for CLI to validate what is a valid value for resource.